.github/workflows/ci.yaml

```diff
@@ -9,12 +9,23 @@ jobs:
     strategy:
       matrix:
         node: [10, 12, 14, 16]
+        include:
+          # use latest npm by default
+          - npm-version: latest
+          # overrides for node 10 and 12
+          - node: 10
+            npm-version: ^7
+          - node: 12
+            npm-version: ^8
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
       - run: node --version
+      # TODO: remove once node 14 is dropped
+      - name: Upgrade to latest npm to support lockfile v2
+        run: npm install -g npm@${{ matrix.npm-version }}
```

Comment:
Without this, npm was failing for older node versions which come with npm<7 and claiming `@opentelemetry/sdk-node` wasn't installed.

Reply:
Moved these changes to https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/448, will rebase after it's merged.

Reply:
Looks like you reviewed the other PR. The comment is correct, it's that node14 comes with npm 6. However npm@latest still supports node14, so the default value in the matrix `latest` works fine